pydatalab/src/pydatalab/routes/v0_1/collections.py

```diff
@@ -419,3 +419,36 @@ def add_items_to_collection(collection_id):
         )
 
     return (jsonify({"status": "success"}), 200)
+
+
+@COLLECTIONS.route("/collections/<collection_id>/items", methods=["DELETE"])
+def remove_items_from_collection(collection_id):
```

Comment:
It would be good to have a quick test of this endpoint! I think until now, I've been using `/save-item` to remove an item from a collection, but this is not very elegant and handling it this way would be much better. Could we also use this endpoint on the item page when removing a collection from the list of collections that a sample is in?